.gitignore

```diff
@@ -12,3 +12,5 @@
 
 # Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
 .glide/
+
+.idea/
```

Comment:
Would you please add a line ending here?

Reply:
Done